web/app/src/ui/menu.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
We prefer to name files just like the component they contain (e.g. `Menu` in this case)